_dev/src/assets/scss/base/_common.scss

```diff
@@ -35,6 +35,10 @@ a.without-hover {
   color: $grey_muted;
 }
 
+.btn-link-muted {
```

Comment:
Maybe `text-color-primary` or `text-color-primary-800` will be more likely to be used than `btn-link-muted`